tests/10-rdfc-create.js

```diff
@@ -30,188 +30,189 @@ const {match} = endpoints.filterByTag({
-            let issuedVc;
-            let proofs;
-            const verificationMethodDocuments = [];
-            before(async function() {
-              issuedVc = await createInitialVc({
-                issuer,
-                vc: credentials.create.document
+  for(const vcVersion of vectors.vcTypes) {
+    describe(`ecdsa-rdfc-2019 (issuers) VC ${vcVersion}`, function() {
+      this.matrix = true;
+      this.report = true;
+      this.implemented = [];
+      this.rowLabel = 'Test Name';
+      this.columnLabel = 'Implementation';
+      for(const [name, {endpoints: issuers}] of match) {
+        // test for each support key type
+        for(const keyType of vectors.keyTypes) {
+        // loop through each issuer in suite
+          for(const issuer of issuers) {
+            // does the endpoint support this test?
+            if(!endpointCheck({endpoint: issuer, keyType, vcVersion})) {
+              continue;
+            }
+            // add implementer name and keyType to test report
+            this.implemented.push(`${name}: ${keyType}`);
+            describe(`${name}: ${keyType}`, function() {
+              let issuedVc;
+              let proofs;
+              const verificationMethodDocuments = [];
+              before(async function() {
+                issuedVc = await createInitialVc({
+                  issuer,
+                  vc: credentials.create[vcVersion].document
+                });
+                // VCs can have multiple proofs so account for that
+                proofs = Array.isArray(issuedVc?.proof) ? issuedVc.proof :
+                  [issuedVc?.proof];
+                const verificationMethods = proofs.map(
+                  proof => proof.verificationMethod);
+                for(const verificationMethod of verificationMethods) {
+                  const verificationMethodDocument = await documentLoader({
+                    url: verificationMethod
+                  });
+                  verificationMethodDocuments.push(verificationMethodDocument);
+                }
               });
-              // VCs can have multiple proofs so account for that
-              proofs = Array.isArray(issuedVc?.proof) ? issuedVc.proof :
-                [issuedVc?.proof];
-              const verificationMethods = proofs.map(
-                proof => proof.verificationMethod);
-              for(const verificationMethod of verificationMethods) {
-                const verificationMethodDocument = await documentLoader({
-                  url: verificationMethod
+              /*
+              @link https://w3c.github.io/vc-di-ecdsa/
+              #verify-derived-proof-ecdsa-sd-2023:~:text=
+              The%20type%20property%20MUST%20be%20DataIntegrityProof
+              */
+              it('The (proof) type property MUST be DataIntegrityProof.',
+                function() {
+                  this.test.cell = {
+                    columnId: `${name}: ${keyType}`, rowId: this.test.title
```

Comment:
so is the `beforeEach` setting these programmatically going to be a in a later PR?

Reply:
On the verifier tests they are (and anything using the helper). Issuer tests havent been ported yet. I ran into some quirks and am out of time to further work on it for now (it's a fairly large design shift). 

The current helper does work with the issuer tests, however the handling of test vectors is distinct enough that it needs more design love to consolidate.